### PR TITLE
Support Is /not Null filter for value is None on webui

### LINKED ISSUE #26427: Can not get task which status is null
### Apache Airflow version

Other Airflow 2 version

### What happened

with List Task Instance airflow webUI,when we search the task which state is null,the result is:no records found.

### What you think should happen instead

should list the task which status is null

### How to reproduce

use airflow webui
List Task Instance
add filter
state equal to null

### Operating System

oracle linux

### Versions of Apache Airflow Providers

2.2.3

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

_No response_

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by chenglongyan**
Snapshot:

------

image

closes: #26427
related: #26461

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
looks like some tests are failing?

**Comment by ashb** [comment]
From a "SQL" point of view `""` (emptry string) is different to `None`, (SQL `NULL`). As you've written your patch it's not possible to find values with an empty string.

Is it worth adding two extra filters for "is null" and "is not null"?

**Comment by chenglongyan** [comment]
Agreed, there are ambiguities...



> From a "SQL" point of view `""` (emptry string) is different to `None`, (SQL `NULL`).

**Review by ashb — APPROVED** [review]

File: airflow/www/static/js/main.js [hunk 1]
```diff
@@ -226,10 +255,14 @@ $(document).ready(() => {
   $.fn.datetimepicker.defaults.sideBySide = true;
   $('.datetimepicker').datetimepicker();
 
+  $('.filters .select2-chosen').each((idx, el) => { filterOpSelected(el); });
```
Comment (bbovenzi) [inline airflow/www/static/js/main.js]:
```suggestion
  $('.filters .select2-chosen').each((idx, elem) => { filterOpSelected(elem); });
```
Looks like `el` is already defined in the file and causing linting issues.

**Review by bbovenzi — APPROVED** [review]

**Pull request merged**